client/src/pages/private/ParticipantTable.js

```diff
@@ -233,46 +191,10 @@ const ParticipantTable = () => {
     );
   }, [isMoH, isSuperUser, roles]);
 
-  // Fetch Data
   useEffect(() => {
-    const getParticipants = async () => {
-      if (!columns) return;
-      if (!selectedTab) return;
-      setLoadingData(true);
-      const { data, pagination: newPagination } = await fetchParticipants(
-        pagination.page * pageSize,
-        filter.locationFilter?.value || '',
-        filter.fsaFilter?.value || '',
-        filter.lastNameFilter?.value || '',
-        filter.emailFilter?.value || '',
-        order.field,
-        order.direction,
-        siteSelector,
-        selectedTabStatuses,
-        filter.isIndigenousFilter?.value
-      );
-      participantsDispatch({
-        type: ParticipantsContext.types.UPDATE_PAGINATION,
-        payload: newPagination,
-      });
-      const newRows = filterData(data, columns);
-      setRows(newRows);
-      setLoadingData(false);
-    };
-
-    getParticipants();
-  }, [
-    fetchParticipants,
-    pagination.page,
-    siteSelector,
-    filter,
-    order,
-    roles,
-    columns,
-    selectedTab,
-    selectedTabStatuses,
-    participantsDispatch,
-  ]);
+    fetchParticipants();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I gather the linter is complaining here, so this feels like a bit of a smell. Have you looked into resolving this hook dependency issue?